MapboxGeocoder/MBGeocodeOptions.swift

```diff
@@ -46,6 +46,11 @@ public class GeocodeOptions: NSObject {
      */
     public var allowedRegion: RectangularRegion?
     
+    /**
+     Limit the number of results returned. The default is `5` for forward geocoding and `1` for reverse geocoding.
+     */
+    public var resultLimit: UInt?
```

Comment:
The reason I codify the defaults that we already support on the server side is in case those change before we can sync up this client code or people can upgrade, so that the docs we provide are still accurate.